process_jartic_simple_v2.py

```
@@ -0,0 +1,205 @@
+                                    
+                                    out_file.write(
+                                        f"{timestamp.isoformat()},JARTIC_{location_code},{location_name},"
+                                        f"0,0,traffic_volume,{traffic_volume},vehicles/5min,{prefecture}\n"
+                                    )
+                                    
+                                    measurements_count += 1
+                                    progress_bar.update(1)
+                                    
+                                    # Update file size every 100k measurements
+                                    if measurements_count % 100000 == 0:
+                                        if output_path.exists():
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)